helm/test-infra/install-helm-via-docker.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+
+which docker > /dev/null
+if [ "$?" != "0" ]; then
+	echo "Error: Docker not found"
+	exit 1
+fi
+if [ "$KUBE_DIR" == "" ]; then
+	echo "Error: Please specify KUBE_DIR"
+	exit 1
+fi
+if [ "$HELM_VERSION" == "" ]; then
+	echo "Error: Please specify HELM_VERSION"
+	exit 1
+fi
+if [ "$HELM_HOME" == "" ]; then
+	HELM_HOME=~/.helm
+fi
+if [ "$KUBECTL_VERSION" == "" ]; then
+	KUBECTL_VERSION=latest
+fi
+
+echo "HELM_HOME=${HELM_HOME}"
+
+docker version
+
+DOCKER_KUBECTL="docker run -i --rm \
+	-v ${KUBE_DIR}:/root/.kube \
+	lachlanevenson/k8s-kubectl:latest"
```

Comment:
I'm a bit uneasy about using this random container image from "the internetz" for kubectl.
May I suggest we use something more official, like Kubernetes' own images?
For example `k8s.gcr.io/hyperkube:v1.13.3` ([Docs](https://github.com/kubernetes/kubernetes/tree/master/cluster/images/hyperkube))

Reply:
Good point.
I wasn't aware of official containers as I only searched through Docker Hub.

I will look into replacing these. 👍

Reply:
Fixed.